### PR TITLE
added CRUD operation for in ItemDaoJdbcImpl.java

### PR DESCRIPTION
**Pull request by Mykhaylo12**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/controller/MainMenu.java [hunk 1]
```diff
@@ -6,10 +6,17 @@
...
+import mate.academy.internetshop.lib.Inject;
+
 public class MainMenu extends HttpServlet {
+    @Inject
+    private static ItemDao itemDao;
+
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        itemDao.get(1L);
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/controller/MainMenu.java]: This is for test, I suppose. please, remove

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        }
+    }
+
+    @Override
+    public boolean delete(Item item) {
+        String query = String.format("delete from %s.items where item_id=%d;",
+                DB_NAME, item.getItemId());
+        try (Statement stmt = connection.createStatement()) {
+            int res = stmt.executeUpdate(query);
+            LOGGER.info(res + " row(s) was effected");
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]:
```suggestion
            LOGGER.info(res + " row(s) was(were) effected");
```

Comment (Mykhaylo12) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: ok

File: src/main/java/mate/academy/internetshop/controller/MainMenu.java (hunk 1 above)
Comment (Mykhaylo12) [reply src/main/java/mate/academy/internetshop/controller/MainMenu.java]: ok

**Review by Tarabarnik — APPROVED** [review]

**Pull request merged**